packages/render-fidelity-tools/test/config.json

```diff
@@ -374,6 +374,16 @@
         "dspbr-pt"
       ]
     },
+    {
+      "name": "khronos-DragonAttenuation",
```

Comment:
Also, this model is pretty (in my opinion), so let's put it toward the top.

Reply:
Agreeeed! I love it myself!